helm/prometheus-rules/templates/platform/cabbage/alerting-rules/cilium.rules.yml

```
@@ -59,4 +59,22 @@ spec:
         severity: page
         team: cabbage
         topic: cilium
-
+    {{- if eq .Values.managementCluster.provider.flavor "capi" }}
+    - alert: FluxHelmReleaseFailed
+      annotations:
+        description: |-
+          {{`Flux HelmRelease {{ $labels.name }} in ns {{ $labels.exported_namespace }} on {{ $labels.installation }}/{{ $labels.cluster_id }} is stuck in Failed state.`}}
```

For maximum convenience lets format the mc-wc string as formatted by teleport for direct copy and paste :)

```suggestion
          {{`Flux HelmRelease {{ $labels.name }} in ns {{ $labels.exported_namespace }} on {{ $labels.installation }}-{{ $labels.cluster_id }} is stuck in Failed state.`}}
```